.github/workflows/release.yml

```diff
@@ -113,3 +139,11 @@ jobs:
               tag_name: `${TAG_NAME}`,
               name: `${TAG_NAME}`
             });
+
+      - name: Trigger chart update
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.HELM_CHART_REPO_ACCESS_TOKEN }}
```

Comment:
this seems to need `content:write` for the helm charts repo, which makes me uneasy. Should be pin the GHA to a specific commit at least?

Reply:
Done